### PR TITLE
doc: manage backup plugin in fleet

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

the doc for #381

To support Kurator's unified backup, restore, and migration features, it's imperative to first configure the backup plugin for Fleet,

This guide will walk you through configuring the Velero-based backup plugin for Fleet, laying the groundwork for Kurator's unified backup, restore, and migration capabilities

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by Xieql** [comment]
I am not sure if it is suitable to add a pic here, because we already have one at "docs/proposals/backup/image"

File: docs/content/en/docs/fleet-manager/backup-plugin.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+```
+
+> **Note**: The `endpoint`(http://172.18.255.200:9000) in `fleet-minio.yaml` is depends on your minio service ip, check your minio service ip in [Minio installation guide](/docs/setup/install-minio).
+
+### Use OBS
+
+If you choose to use OBS, apply the following configuration example:
+
+```console
+kubectl apply -f examples/backup/fleet-obs.yaml
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup-plugin.md:81]: You can try to remove the file

```
kubectl apply -f - <<EOF

EOF
````

File: docs/content/en/docs/fleet-manager/backup-plugin.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      kind: AttachedCluster
+  plugin:
+    backup:
+      storage:
+        location:
+          bucket: kurator-obs
+          provider: huaweicloud
+          endpoint: http://obs.cn-south-1.myhuaweicloud.com
+          region: cn-south-1
+        secretName: obs-credentials
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup-plugin.md:108]: any guide on the secret format?

File: docs/content/en/docs/fleet-manager/backup-plugin.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+```
+
+Let's delve into the `spec` section of the above Fleet:
+
+- `clusters`: Contains the two `AttachedCluster` objects created earlier, indicating that the backup plugin will be installed on these two clusters.
+  
+- `plugin`: The `backup` indicates the description of a backup plugin. Currently, it only contains configurations related to storage. For more configuration options, please refer to the [Fleet API](https://kurator.dev/docs/references/fleet-api/).
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup-plugin.md:115]: s/configurations related to storage/storage related configurations

File: docs/content/en/docs/fleet-manager/backup-plugin.md (hunk 2 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup-plugin.md:108]: already consider it, not sure if it is enough:

line55: For more configuration options, please refer to the [Fleet API](https://kurator.dev/docs/references/fleet-api/).

the content of Fleet API link:

image

File: docs/content/en/docs/fleet-manager/backup-plugin.md (hunk 1 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup-plugin.md:81]: o'kok

**Comment by Xieql** [comment]
@hzxuzhonghu what about this backup arch digram

File: docs/content/en/docs/fleet-manager/backup-plugin.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+
+## Prerequisites
+
+### 1. Fleet Manager Setup
+
+Set up the Fleet manager by following the instructions in the [installation guide](/docs/setup/install-fleet-manager/).
+
+### 2. Configuring Object Storage for Velero
+
+Kurator's backup relies on Velero. Hence, [Object Storage](https://velero.io/docs/v1.12/how-velero-works/) is essential to restore Kubernetes resources. We support various object storage options. In this guide, we provide detailed usage instructions for the following configurations:
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/fleet-manager/backup-plugin.md:36]: is essential to store

**Review by hzxuzhonghu — APPROVED** [review]

File: docs/content/en/docs/fleet-manager/backup-plugin.md (hunk 4 above)
Comment (Xieql) [reply docs/content/en/docs/fleet-manager/backup-plugin.md]: fixed

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**